src/translib/transformer/host_acct_mgr.go

```diff
@@ -15,12 +15,8 @@ func roleToGroup(role string) []string {
 	switch role {
 	case "admin":
 		return []string{"admin", "sudo", "docker"}
-	
-	case "operator":
-		return []string{"operator", "docker"}
-	
 	default:
-		return []string{}
+		return []string{"operator", "docker"}
```

Comment:
Don't want to assign users by default to the `operator` group. Rather, they should be assigned to the same group as the provided role (plus `docker`).

i.e., if I configure `username jeffyin password <password> role xyz123`, I should, as a result, be assigned to the `xyz123` and `docker` groups. The system (currently) would then treat me as a non-admin user.

Reply:
Hamd does not accept any role other than admin and operator.